atlas/AtlasMl/tests/test_ai_agent.py

```
@@ -0,0 +1,262 @@
+import pytest
+from unittest.mock import AsyncMock, patch, MagicMock
+
+from tests.conftest import mock_generate_embeddings_openai, mock_weaviate_client
```

Ruff (0.12.2)

`tests.conftest.mock_generate_embeddings_openai` imported but unused

Remove unused import

(F401)